.editorconfig

```diff
@@ -0,0 +1,10 @@
+# top-most EditorConfig file
+root = true
+
+[*.{cs,fs,fsx}]
+indent_size = 4
+indent_style = space
+
+[*.{sln,csproj,fsproj,config,xml}]
+indent_size = 2
```

Comment:
I used the current repository state as a style source. All the specified files currently have 2 spaces indentation. I decided to not change that and just explicitly describe that in configuration :pensive:

AFAIK, 2 spaces is Visual Studio default for `sln`, `csproj`, `fsproj` and `packages.config`, so we shouldn't override it.